### DISCUSSION TITLE
Add context.isEditor to the docs

### DISCUSSION THREAD

**Discussion by anastasiauz** [OFFLINE-GmbH/oc-boxes-support#101] [Ideas and feature requests]
Thank you for the brilliant plugin, October for years missed smth like this! I would like to suggest adding to the docs information about {{ context.isEditor }} availability in the partial. Sometimes it's useful to show smth else on the backend editor, instead of the box contents or adding smth to it, and it can be done easily by checking the context (but it is not mentioned anywhere in the docs). Thanks!

**Comment by tobias-kuendig** [comment]
Good point! Thank you @anastasiauz, this has been added to the docs:

https://docs.boxes.offline.ch/concepts/box-partials.html#check-if-the-partial-is-rendered-in-the-editor
